src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj

```diff
@@ -1,197 +1,93 @@
-    <Compile Include="Core\DataStoreChangedTrigger.cs" />
-    <Compile Include="Core\DataTrigger.cs" />
-    <Compile Include="Core\ExtendedVisualStateManager.cs" />
-    <Compile Include="Core\GoToStateAction.cs" />
-    <Compile Include="Core\ICondition.cs" />
-    <Compile Include="Core\LaunchUriOrFileAction.cs" />
-    <Compile Include="Core\PropertyChangedTrigger.cs" />
-    <Compile Include="Core\PrototypingActions.cs" />
-    <Compile Include="Core\RemoveElementAction.cs" />
-    <Compile Include="Core\SetDataStoreValueAction.cs" />
-    <Compile Include="Core\TimerTrigger.cs" />
-    <Compile Include="CustomPropertyValueEditorAttribute.cs" />
-    <Compile Include="DataBindingHelper.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="DefaultTriggerAttribute.cs" />
-    <Compile Include="DependencyObjectHelper.cs" />
-    <Compile Include="EventObserver.cs" />
-    <Compile Include="EventTrigger.cs" />
-    <Compile Include="EventTriggerBase.cs" />
-    <Compile Include="ExceptionStringTable.Designer.cs">
-      <SubType>Code</SubType>
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>ExceptionStringTable.resx</DependentUpon>
-    </Compile>
-    <Compile Include="IAttachedObject.cs" />
-    <Compile Include="Input\KeyTrigger.cs" />
-    <Compile Include="Input\TranslateZoomRotateBehavior.cs" />
-    <Compile Include="Interaction.cs" />
-    <Compile Include="InteractionContext.cs" />
-    <Compile Include="InvokeCommandAction.cs" />
-    <Compile Include="ITickTimer.cs" />
-    <Compile Include="Layout\FluidMoveBehavior.cs" />
-    <Compile Include="Layout\MouseDragElementBehavior.cs" />
-    <Compile Include="Media\PlaySoundAction.cs" />
-    <Compile Include="Media\StoryboardAction.cs" />
-    <Compile Include="Media\StoryboardTrigger.cs" />
-    <Compile Include="Media\TransitionEffect.cs" />
-    <Compile Include="NameResolver.cs" />
-    <Compile Include="ScreenType.cs" />
-    <Compile Include="Serializer.cs" />
-    <Compile Include="TargetedTriggerAction.cs" />
-    <Compile Include="TriggerAction.cs" />
-    <Compile Include="TriggerActionCollection.cs" />
-    <Compile Include="TriggerBase.cs" />
-    <Compile Include="TriggerCollection.cs" />
-    <Compile Include="TypeConstraintAttribute.cs" />
-    <Compile Include="TypeConverterHelper.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="VisualStateUtilities.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="..\RuntimeVersion.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="..\Version\Version.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <EmbeddedResource Include="ExceptionStringTable.resx">
-      <Public>True</Public>
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>ExceptionStringTable.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net45'">
```

Comment:
do we need this Condition if targeting only net45?

Reply:
Aaah good catch. No we don't need it … yet :-) That accidentally came from my .NET Core 3.0 port